### PR TITLE
add amqp retry conf

### PR DESCRIPTION
**Pull request by AlvaroVega**
reopen previous PR

### CODE REVIEW

File: lib/configService.js [hunk 1]
```diff
@@ -51,6 +51,8 @@ function processEnvironmentVariables() {
             'IOTA_AMQP_EXCHANGE',
             'IOTA_AMQP_QUEUE',
             'IOTA_AMQP_DURABLE',
+            'IOTA_AMQP_RETRIES',
```
Comment (fgalan) [inline lib/configService.js]: Just to check...

This variables has been already include in documentation in a past PR?

File: lib/bindings/AMQPBinding.js [hunk 2]
```diff
@@ -78,6 +78,20 @@ function start(callback) {
         queue = constants.AMQP_DEFAULT_QUEUE;
```
Comment (fgalan) [inline lib/bindings/AMQPBinding.js]: CNR entry needed?

Probably not, but just checking...

Comment (AlvaroVega) [reply lib/bindings/AMQPBinding.js]: CNR is included in previous PR

File: lib/configService.js (hunk 1 above)
Comment (AlvaroVega) [reply lib/configService.js]: just added

File: docs/installationguide.md [hunk 3]
```diff
@@ -87,6 +87,8 @@ IoT Agent. The following attributes are accepted:
 * **exchange**: Exchange in the AMQP broker
 * **queue**: Queue in the AMQP broker
 * **durable**: durable queue flag (default is `false`).
+* **retries**: Number of AMQP connection errror retries (default is 5).
+* **retryTime**: Time between AMQP connection retries (default is 5'').
```
Comment (fgalan) [inline docs/installationguide.md]: `5''`

Typo?

Comment (AlvaroVega) [reply docs/installationguide.md]: 5'' == 5 seconds

Comment (fgalan) [reply docs/installationguide.md]: We typically use the style "5 seconds" along all documentation. However, we can also use 5'' (personally, I find it a bit confusing, but it's fine).

**Pull request merged**